### PR TITLE
XS✔ ◾ Fix spell errors

### PR DESCRIPTION
**Pull request by karl-johan-grahn**
Fix doc spelling errors

### CODE REVIEW

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull stakater/workshop-exercise:SNAPSHOT-PR-69-37c3a977`

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull stakater/workshop-exercise:SNAPSHOT-PR-69-05062d6e`

File: docs/2-attack-of-the-pipelines/3b-tekton.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ The task takes parameter needed to build and push the image as parameters, such
 
 ![image-push-script](./images/image-push-script.png)
 
-#### 4 - stakater-helm-push-v1 🅿
+#### 4 - `stakater-helm-push-v1` 🅿
```
Comment (callum-stakater) [inline docs/2-attack-of-the-pipelines/3b-tekton.md:90]: not this one, not sure why its being set like this, it is not consistent with the  rest of items in this list

File: docs/2-attack-of-the-pipelines/3b-tekton.md [hunk 2]
```diff
@@ -256,7 +256,7 @@ With all these components in place - now it's time to trigger pipeline via webho
 
     ![updated-Nordmart-apps-GitOps-config](images/updated-nordmart-apps-gitops-config.png)
 
-  > For pushes to `main` branch, your application is automagically updated in the `<TENANT_NAME>-dev` namespace.
+  > For pushes to `main` branch, your application is automatically updated in the `<TENANT_NAME>-dev` namespace.
```
Comment (callum-stakater) [inline docs/2-attack-of-the-pipelines/3b-tekton.md:259]: ;(

File: docs/2-attack-of-the-pipelines/3b-tekton.md (hunk 1 above)
Comment (karl-johan-grahn) [reply docs/2-attack-of-the-pipelines/3b-tekton.md:90]: fixed other headings

File: docs/2-attack-of-the-pipelines/3b-tekton.md (hunk 2 above)
Comment (karl-johan-grahn) [reply docs/2-attack-of-the-pipelines/3b-tekton.md:259]: either update vocabulary or fix it for now, for now I just use accepted word

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull stakater/workshop-exercise:SNAPSHOT-PR-69-11535986`

**Review by callum-stakater — APPROVED** [review]
Beautiful! Thanks!

**Pull request merged**